main.go

```diff
@@ -18,7 +18,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	changes, err := diff.Changes(os.Args[1], os.Args[2])
+	var ign []string
```

Comment:
This is kind of ghetto, but `main.go` is actually only used for tests -- locutus has its own wrapper around the `diff` package which I'll just hardcode to ignore `.git`.